test/integration/targets/vault/single_vault_as_string.yml

```
@@ -0,0 +1,109 @@
+- hosts: localhost
+  vars:
+    vaulted_valued: !vault |
+          $ANSIBLE_VAULT;1.1;AES256
+          35323961353038346165643738646465376139363061353835303739663538343266303232326635
+          3365353662646236356665323135633630656238316530640a663362363763633436373439663031
+          33663433383037396438656464636433653837376361313638366362333037323961316364363363
+          3835616438623261650a636164376534376661393134326662326362323131373964313961623365
+          3833
+  tasks:
+    - debug:
+        msg: "{{ vaulted_valued }}"
+
+    - debug:
+        msg: "{{ vaulted_valued|type_debug }}"
+
+    - assert:
+        that:
+          - vaulted_valued is vault_encrypted
```

Super minor, but `vaulted_value` seems to read better.
```suggestion
          - vaulted_value is vault_encrypted
```

---

You caught me being lazy. I typoed it, and then just kept copying it.